modules/nw-udn-additional-config-details.adoc

```diff
@@ -21,6 +21,13 @@ The following table explains additional configurations for `ClusterUserDefinedNe
 
 The `joinSubnets` field configures the routing between different segments within a user-defined network. Dual-stack clusters can set 2 subnets, one for each IP family; otherwise, only 1 subnet is allowed. This field is only allowed for the `Primary` network.
 
+|`spec.network.<topology>.excludeSubnets`
+|`spec.<topology>.exlcudeSubnets`
+|string
+|Specifies a list of CIDRs to be removed from the specified CIDRs in the `subnets` field. The CIDRs in this list must be in range of at least one subnet specified in `subnets`. When omitted, no IP addresses are excluded, and all IP addresses specified in the `subnets` field are subject to assignment. You must use standard CIDR notation. For example, `10.128.0.0/16`. This field must be omitted if the `subnets` field is not set or if the `ipam.mode` field is set to `Disabled`.
+
+When the `physicalNetworkName` field  points to OVS bridge mapping of a network with reserved IP addresses that are not assigned by OVN-Kubernetes, the specified CIDRs will not be assigned. For example, if `subnets: "10.0.0.0/24"` are set and `excludeSubnets: "10.0.0.200/30" are as well, the following addresses will not be assigned to pods: `10.0.0.201`, `10.0.0.202`.
```

Comment:
This paragraph reads a bit strange. It kind of implies the `physicalNetworkName` is relevant here.

IMHO, it is not. What matters is the user's intention and/or the services (not speaking about Kubernetes services here ...) they have deployed in their physical network - whatever IPs there are in the physical network must be explicitly listed in the exclude subnets, to prevent IP duplicates in the subnet.

Reply:
Updated. Does this work? @maiqueb